src/views/darkroom.c

```diff
@@ -3323,7 +3323,7 @@ void init_key_accels(dt_view_t *self)
   dt_accel_register_view(self, NC_("accel", "image back"), GDK_KEY_BackSpace, 0);
 
   // toggle ISO 12646 color assessment condition
-  dt_accel_register_view(self, NC_("accel", "color assessment"), GDK_KEY_b, GDK_CONTROL_MASK);
```

Comment:
Given last comment I suppose this last commit should be removed as CTRL+B is ok.